examples/ChatQnA/ChatQnA_Guide.rst

```diff
@@ -3,89 +3,182 @@
+The core of this architecture are vector databases, which are instrumental in
+enabling efficient and semantic retrieval of information. These databases store
+data as vectors, allowing RAG to swiftly access the most pertinent documents or
+data points based on semantic similarity.
+
+Central to the RAG architecture is the use of a generative model, which is
+responsible for generating responses to user queries. The generative model is
+trained on a large corpus of customized and relavent text data and is capable of
+generating human-like responses. Developers can easily swap out the generative
+model or vector database with their own custom models or databases. This allows
+developers to build chatbots that are tailored to their specific use cases and
+requirements. By combining the generative model with the vector database, RAG
+can provide accurate and contextually relevant responses speicfic to your users'
+queries.
+
+The ChatQnA example is designed to be a simple, yet powerful, demonstration of
+the RAG architecture. It is a great starting point for developers looking to
+build chatbots that can provide accurate and up-to-date information to users.
+
+In addition, the ChatQnA provides several deployment options, including
+single-node on-premises deployments using Xeon Scalable Processors, Gaudi
+servers, NVIDIA GPUs, and AI PCs. It also supports Kubernetes deployments with
+and without the GenAI Management Console (GMC), as well as cloud-native
+deployments using Red Hat OpenShift Container Platform (RHOCP).
+
+Preview
 =======
 
-AI catalog if applicable, or recorded demos. 
+To get a preview of the ChatQnA example, visit the
+`AI Explore site <https://aiexplorer.intel.com/explore>`_. The **ChatQnA Solution**
+provides a basic chatbot while the **ChatQnA with Augmented Context**
+allows you to upload your own files in order to quickly experiment with a RAG
+solution to see how a developer supplied corpus can provide relevant and up to
+date responses.
 
-Key Implementation Details 
+Key Implementation Details
 ==========================
 
-Tech Overview
-*************
+Embedding:
+  The process of transforming user queries into numerical representations called
+  embeddings.
+Vector Database:
+  The storage and retrieval of relevant data points using vector databases.
+RAG Architecture:
+  The use of the RAG architecture to combine knowledge bases and generative
+  models for development of chatbots with relevant and up to date query
+  responses.
+Large Language Models (LLMs):
+  The training and utilization of LLMs for generating responses.
+Deployment Options:
+  production ready deployment options for the ChatQnA
+  example, including single-node deployments and Kubernetes deployments.
 
 How It Works
 ============
 
-High level graphics to summarize the application.
+The ChatQnA Examples follows a basic flow of information in the chatbot system,
+starting from the user input and going through the retrieve, analyze, and
```

Comment:
do we want to use "Re-ranker" instead of "analyze" to make component mental mapping easier?